### DISCUSSION TITLE
Updating the .mjmlconfig file

### DISCUSSION THREAD

**Discussion by DanaIntouch** [mjmlio/mjml#2494] [Q&A] [answered]
OK, I understand Terminal (Mac) to an extent.

I need to update my config file with the mj-list package (installed -g like mjml). 

I feel dumb asking this but...could someone help me figure this out?

Thank you!

**Comment by iRyusa** [comment, marked as answer]
You should not install mjml globally as it could be a security issue. Just use a local install inside a folder with a package.json

**Reply by DanaIntouch** [reply]
Thank you.

I have installed -g. I was trying to insert the mj-list add on...it isn't necessary. I needed to build mjtables for what I wanted.

mj-list might be deprecated now, anyway!

Love this mjml thingy.
